pytypes/stubfile_2_converter.py

```diff
@@ -147,12 +147,12 @@ def convert(in_file, out_file = None):
 		os.makedirs(directory)
 
 	with open(out_file, 'w') as out_file_handle:
-		lines = [""""",
+		lines = ["'''",
```

Comment:
I didn't mean you to revert it. Actually the generated file should have PEP 8 compliant docstring.
Just `"""""` wasn't suitable. Should have been `'"""'`
(Same below)